packages/spaces/commands/vpn/connect.js

```diff
@@ -32,13 +32,15 @@ module.exports = {
   topic: 'spaces',
   command: 'vpn:connect',
   description: 'create VPN',
-  help: `Example:
+  help: `Private Spaces can be connected to another private network via an IPSec VPN connection allowing dynos to connect to hosts on your private networks and vice versa.
+The connection is established over the public Internet but all traffic is encrypted using IPSec.
+  
+  Example:
```

Comment:
can you move this to a top-level examples property? That will render it a bit more nicely. Here is an example of what I mean: https://github.com/heroku/cli/blob/43ccdbbfbf1d2fe189d2227ed7b0e6b5162a99cc/packages/pipelines-v5/commands/pipelines/destroy.js#L11